querybook/webapp/components/EnvironmentAppSidebar/EnvironmentAppSidebar.tsx

```diff
@@ -19,9 +20,12 @@ import { matchKeyMap, KeyMap } from 'lib/utils/keyboard';
 import './EnvironmentAppSidebar.scss';
 
 const SIDEBAR_WIDTH = 320;
+const EMBED_PATH_STRING = 'embedded';
 
 export const EnvironmentAppSidebar: React.FunctionComponent = () => {
-    const [collapsed, setCollapsed] = React.useState(false);
+    const location = useLocation();
+    const isEmbed = location.pathname.includes(EMBED_PATH_STRING);
+    const [collapsed, setCollapsed] = React.useState(isEmbed);
```

Comment:
also i think its cleaner if you check `/_/` is in pathname instead.

```
React.useState(() => location.pathname.includes(EMBED_PATH_STRING));

Reply:
Updated